examples/angular2/services/store.ts

```diff
@@ -0,0 +1,76 @@
+export class Todo {
+	completed: Boolean;
+	editing: Boolean;
+	title: String;
+	uid: String;
+	cleanTitle(title:String) {
+		return title.trim();
+	}
+	setTitle(title:String) {
```

Comment:
Can we have a space after `:`? This is what I've seen in most projects (and other languages that have annotations like this; Scala etc).